tilequeue/metatile.py

```diff
@@ -16,6 +18,10 @@ def make_single_metatile(size, tiles):
     if len(tiles) == 0:
         return []
 
+    if date_time is None:
+        from time import gmtime
```

Comment:
:laughing: If we want to get more opinionated about imports, I personally like to have just 1 import  per line and sort them alphabetically. 1 import per line makes it easier to search for, and sorting alphabetically just takes all the thinking out of it. But, I'm with you that I'm happy the hateful flake8 doesn't complain about it ;)